release notes/v0.26.0.md

```diff
@@ -0,0 +1,122 @@
+k6 v0.26.0 is here! :tada:
+
+This release contains mostly bug fixes, though it also has several new features and enhancements! They include a new JS compatibility mode option, exporting the end-of-test summary to a JSON report file, speedups to the InfluxDB and JSON outputs, a brand new CSV output, multiple layered HTTP response body decompression, being able to use `console` in the init context, a new optional column in the summary, and Docker improvements!
+
+
+Thanks to  @Sirozha1337, @openmohan, @MMartyn, @KajdeMunter, @dmitrytokarev and @dimatock for contributing to this release!
+
+## New features and enhancements!
+
+### A new JavaScript compatibility mode option (#1206)
+
+This adds a way to disable the automatic script transformation by [Babel](https://babeljs.io/) (v6.4.2) and [core-js (v2)](https://github.com/zloirock/core-js) polyfills bundled in k6. With the new `base` compatibility mode, k6 will instead rely only on the [goja](https://github.com/dop251/goja/) runtime and what is built into k6.
```

Comment:
```suggestion
This adds a way to disable the automatic script transformation the by [Babel](https://babeljs.io/) (v6.4.2) and [core-js (v2)](https://github.com/zloirock/core-js) polyfills, bundled in k6. With the new `base` compatibility mode, k6 will instead rely only on the [goja](https://github.com/dop251/goja/) runtime and what is built into k6.
```
the `the` needs to be there, since the whole sentence is `transformation by <whatever>, bundled in k6`